examples/tests/invalid/reassign_to_nonreassignable.w

```diff
@@ -5,17 +5,30 @@ x = x + 1;
 // Assign to non-reassignable field
 resource R {
   f: num;
+  inflight inflight_f: num;
+  
   init() {
     this.f = 1;
   }
+
+  inflight init() {
+    this.inflight_f = 1;
+  }
   
   inc() {
     this.f = this.f + 1;
```

Comment:
Maybe we could also test a case like
`this.child_resource.f = 1 // this.child_resource.f is not reassignable`